docs/guides/mssqlserver/configuration/using-config-file.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Run MSSQLServer with Custom Configuration
+menu:
+  docs_{{ .version }}:
+    identifier: ms-configuration-config-file
+    name: Config File
+    parent: ms-configuration
+    weight: 10
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
+# Using Custom Configuration File
+
+KubeDB supports providing custom configuration for MSSQLServer. This tutorial will show you how to use KubeDB to run SQL Server with custom configuration.
+
+## Before You Begin
+
+- You need to have a Kubernetes cluster, and the kubectl command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+- Now, install KubeDB cli on your workstation and KubeDB operator in your cluster following the steps [here](/docs/setup/README.md). Make sure install with helm command including `--set global.featureGates.MSSQLServer=true` to ensure MSSQLServer CRD installation.
```

Comment:
Either point to an installation where custom installation with featureGates are shown or you just give the installation command sample here.